package.json

```diff
@@ -48,6 +48,7 @@
   "devDependencies": {
     "@types/chai": "^4.1.6",
     "@types/chai-as-promised": "^7.1.0",
+    "@types/cls-hooked": "^4.3.0",
```

Comment:
What's this used for?

Reply:
I get this error without it, didn't investigate further:
```
node_modules/express-http-context/index.d.ts:2:27 - error TS7016: Could not find a declaration file for module 'cls-hooked'. '/home/spider/prog/pisa/node_modules/cls-hooked/index.js' implicitly has an 'any' type.
  Try `npm install @types/cls-hooked` if it exists or add a new declaration (.d.ts) file containing `declare module 'cls-hooked';`

2 import { Namespace } from 'cls-hooked';
```

Reply:
Fair enough